src/components/View.js

```diff
@@ -171,33 +170,35 @@ class View extends Component {
 
   render = () => {
     return (
-      <a-scene physics="debug: false; friction: 3; restitution: .3;" embedded debug="false">
```

Comment:
What's up with these changes?

Reply:
These are just whitespace changes I didn't intentionally make, is there a way to ignore/revert these easily?